test/PowerShellEditorServices.Test/Console/ChoicePromptHandlerTests.cs

```diff
@@ -96,7 +96,7 @@ internal class TestChoicePromptHandler : ChoicePromptHandler
 
         public int TimesPrompted { get; private set; }
 
-        public TestChoicePromptHandler() : base(new NullLogger())
+        public TestChoicePromptHandler() : base(Logging.CreateLogger().Build())
```

Comment:
Yeah, perhaps having a static CreateNullLogger method or something like that would be more clear/explicit